templates/primary-secondary/primary-secondary.js

```diff
@@ -1281,6 +1314,19 @@ class TemplatePrimarySecondary extends LocalizeCoreElement(LitElement) {
 		`;
 
 	}
+
+	#handleFormDirty(e) {
+		this.dispatchEvent(new CustomEvent('d2l-form-dirty', { detail: e.detail }));
+	}
+
+	#handleFormSubmit(e) {
+		this.dispatchEvent(new CustomEvent('d2l-form-submit', { detail: e.detail }));
+	}
+
+	#handleInvalidForm(e) {
+		this._formErrorSummary = this.form.errorSummary;
+		this.dispatchEvent(new CustomEvent('d2l-form-invalid', { detail: e.detail }));
+	}
```

Comment:
Since we need to catch and redispatch the events here anyways, should we change the names? Something like `d2l-primary-secondary-form-dirty`, `d2l-primary-secondary-form-submit`, etc.